yt_dlp/extractor/vrt.py

```diff
@@ -1,82 +1,215 @@
+        vrtPlayerToken = self._download_json(
+            '%s/tokens' % self._REST_API_BASE_TOKEN, video_id,
+            'Downloading token', headers=headers, data=json.dumps({
+                'identityToken': vrtnutoken
+            }).encode('utf-8'))['vrtPlayerToken']
+
+        video_info = self._download_json(
+            '%s/media-items/%s' % (self._REST_API_BASE_VIDEO, video_id),
+            video_id, 'Downloading video JSON', query={
+                'vrtPlayerToken': vrtPlayerToken,
+                'client': 'vrtnu-web@PROD',
+            }, expected_status=400)
+        if 'title' not in video_info:
+            code = video_info.get('code')
+            if code == 'AUTHENTICATION_REQUIRED':
+                self.raise_login_required()
+            elif code == 'INVALID_LOCATION':
+                self.raise_geo_restricted(countries=['BE'])
+            raise ExtractorError(video_info.get('message') or code, expected=True)
+
+        formats = []
+        subtitles = {}
+        for target in video_info['targetUrls']:
+            format_url, format_type = url_or_none(target.get('url')), str_or_none(target.get('type'))
+            if not format_url or not format_type:
+                continue
+            format_type = format_type.upper()
+            if format_type in self._HLS_ENTRY_PROTOCOLS_MAP:
+                fmts, subs = self._extract_m3u8_formats_and_subtitles(
+                    format_url, video_id, 'mp4', self._HLS_ENTRY_PROTOCOLS_MAP[format_type],
+                    m3u8_id=format_type, fatal=False)
+                formats.extend(fmts)
+                subtitles = self._merge_subtitles(subtitles, subs)
+            elif format_type == 'HDS':
+                formats.extend(self._extract_f4m_formats(
+                    format_url, video_id, f4m_id=format_type, fatal=False))
+            elif format_type == 'MPEG_DASH':
+                fmts, subs = self._extract_mpd_formats_and_subtitles(
+                    format_url, video_id, mpd_id=format_type, fatal=False)
+                formats.extend(fmts)
+                subtitles = self._merge_subtitles(subtitles, subs)
+            elif format_type == 'HSS':
+                fmts, subs = self._extract_ism_formats_and_subtitles(
+                    format_url, video_id, ism_id='mss', fatal=False)
+                formats.extend(fmts)
+                subtitles = self._merge_subtitles(subtitles, subs)
+            else:
+                formats.append({
+                    'format_id': format_type,
+                    'url': format_url,
+                })
+
+        subtitle_urls = video_info.get('subtitleUrls')
+        if isinstance(subtitle_urls, list):
+            for subtitle in subtitle_urls:
+                subtitle_url = subtitle.get('url')
+                if subtitle_url and subtitle.get('type') == 'CLOSED':
+                    subtitles.setdefault('nl', []).append({'url': subtitle_url})
 
         return {
```

Comment:
then here's how we can inline everything in the return:

```py
        return {
            **traverse_obj(details, {
                'title': 'title',
                'description': ('description', {clean_html}),
                'timestamp': ('data', 'episode', 'onTime', 'raw', {parse_iso8601}),
                'release_timestamp': ('data', 'episode', 'onTime', 'raw', {parse_iso8601}),
                'series': ('data', 'program', 'title'),
                'season': ('data', 'season', 'title', 'value'),
                'season_number': ('data', 'season', 'title', 'raw', {int_or_none}),
                'season_id': ('data', 'season', 'id', {str_or_none}),
                'episode': ('data', 'episode', 'number', 'value', {str_or_none}),
                'episode_number': ('data', 'episode', 'number', 'raw', {int_or_none}),
                'episode_id': ('data', 'episode', 'id', {str_or_none}),
            }),
            'id': video_id,
            'display_id': display_id,
            'channel': 'VRT',
            'formats': formats,
            'duration': float_or_none(video_info.get('duration'), 1000),
            'thumbnail': url_or_none(video_info.get('posterImageUrl')),
            'subtitles': subtitles,
            '_old_archive_ids': [make_archive_id('VrtNU', video_id)],
        }
```
we should add `_old_archive_ids` since the `VrtNU` extractor is being superseded by this one